demo/src/main.rs

```diff
@@ -21,6 +21,9 @@ include_cpp! {
     generate!("Goat")
 }
 
+#[link(name = "autocxx-demo")]
+extern "C" {}
+
```

Comment:
Still can't do this, sorry.

Reply:
Ping! @bsilver8192 do you plan to address this? Otherwise, I'll get around to it in the end.

Reply:
Sorry, forgot about this. Done